sunshine/video.cpp

```diff
@@ -575,6 +602,7 @@ static std::vector<encoder_t> encoders {
 #endif
 #ifdef _WIN32
   amdvce,
+  quicksync,
```

Comment:
Would this encoder work on Linux and/or Mac as well?

Reply:
Not sure, only tested on Windows. I can test it on linux and mac once I get this working well